Mathlib/RingTheory/Bialgebra/Equiv.lean

```diff
@@ -268,6 +266,18 @@ theorem trans_toBialgHom :
 theorem coe_toEquiv_trans : (e₁₂ : A ≃ B).trans e₂₃ = (e₁₂.trans e₂₃ : A ≃ C) :=
   rfl
 
+@[simp]
+lemma apply_symm_apply (e : A ≃ₐc[R] B) : ∀ x, e (e.symm x) = x := e.toEquiv.apply_symm_apply
+
+@[simp]
+lemma symm_apply_apply (e : A ≃ₐc[R] B) : ∀ x, e.symm (e x) = x := e.toEquiv.symm_apply_apply
+
+@[simp] lemma comp_symm (e : A ≃ₐc[R] B) : (e : A →ₐc[R] B).comp e.symm = .id R B := by ext; simp
+@[simp] lemma symm_comp (e : A ≃ₐc[R] B) : (e.symm : B →ₐc[R] A).comp e = .id R A := by ext; simp
```

Comment:
nit: consider proving via an injective cast to `Equiv` (or `AlgEquiv` / `CoalgEquiv`), and then invoking the lemma of the same name there (to make it obvious that the name is consistent)

bors d+